ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthProtocolManager.java

```diff
@@ -412,43 +413,41 @@ private void handleStatusMessage(final EthPeer peer, final Message message) {
     try {
       if (!status.networkId().equals(networkId)) {
         LOG.atDebug()
-            .setMessage("Mismatched network id: {}, EthPeer {}...")
+            .setMessage("Mismatched network id: {}, peer {}")
             .addArgument(status.networkId())
-            .addArgument(peer.getShortNodeId())
-            .log();
-        LOG.atTrace()
-            .setMessage("Mismatched network id: {}, EthPeer {}")
-            .addArgument(status.networkId())
-            .addArgument(peer)
+            .addArgument(getPeerOrPeerId(peer))
```

Comment:
to avoid doing the computation when debug level is not enabled, in the `addArgument` you need to use a lambda. I cannot make a suggestion since there are deleted lines, but the code should look like:
```
.addArgument(status::networkId)
.addArgument(() -> getPeerOrPeerId(peer))
```

Reply:
I've updated all the places where we do log information to use lambdas where possible.